.circleci/config.yml

```diff
@@ -26,33 +26,30 @@ jobs:
       - run:
           command: go run build/ci.go lint
 
-  build-dockerfile:
+  push-geth:
     docker:
       - image: cimg/base:2022.04
-    parameters:
-      image-name:
-        description: Image name
-        type: string
-      image-tag:
-        description: Image tag
-        type: string
-      target:
-        description: Dockerfile target
-        type: string
-        default: ""
-      dockerfile:
-        description: Dockerfile to use
-        type: string
     steps:
-      - checkout
-      - setup_remote_docker:
-          version: 20.10.12
-      - run:
-          name: Build
-          command: |
-            echo "$DOCKER_PASS" | docker login -u "$DOCKER_USERNAME" --password-stdin
-            docker build -t "ethereumoptimism/<<parameters.image-name>>:<<parameters.image-tag>>" -f <<parameters.dockerfile>> <<#parameters.target>>--target <<parameters.target>><</parameters.target>> .
-            docker push "ethereumoptimism/<<parameters.image-name>>:<<parameters.image-tag>>"
+      - when:
+          condition:
+            or:
+              - equal: [ optimism, <<pipeline.git.branch>> ]
```

Comment:
Almost nothing. We'll still build the Dockerfile with this PR, we just won't push it. I don't want to clutter up our package store too much.

Reply:
The other build is basically a smoke test. Depending on how we use geth, it can be helpful to have prebuilt docker images for each commit during testing. It's easier to push and have it be auto built than building it locally then setting it all up.